src/plugins/fetch/index.ts

```diff
@@ -3,7 +3,7 @@ import { GenericObject } from '../../types/object';
 import router from '../router';
 import { LOGIN } from './routes/mass-api';
 
-export const api_url = import.meta.env.VITE_API_URL as string | undefined || '';
+export const api_url = import.meta.env.SERVER_URL as string | undefined || '';
```

Comment:
@IAmRGroot  What is the difference between accessing a variable through `import.meta.env` and directly accessing `process.env`?

Reply:
Process.env was not accessible in certain vite config files and can import.meta be used everywhere (I think). So same same in this particular example

Reply:
You might be right about the prefix. It also seems they can be inlined, whatever that is ([source](https://github.com/vitejs/vite/issues/1574)). Would that be just referencing them directly as constants, like so: `VITE_SERVER_URL`?